### PR TITLE
add weekly update March 13th

### PR DESCRIPTION
**Pull request by julianduque**
Weekly update ready for review/publish - Related: #41

### CODE REVIEW

File: weekly-updates/weekly-update.2015-03-13.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* [Mikeal Rogers](https://github.com/mikeal) was interviewed for [Descriptive](http://descriptive.audio) podcast on an episoded called [We've Never Had This Many Active Contributors to Core Before](http://descriptive.audio/episodes/12)
+* [Mark Wolfe](https://twitter.com/wolfeidau) gave a [talk about io.js](https://twitter.com/wolfeidau/status/575785856545378304) at [@melbjs](https://twitter.com/melbjs) meetup, slides are published [here](https://speakerdeck.com/wolfeidau/iojs-bringing-es6-to-the-node)
+* [dockeri.co](http://dockeri.co/) now runs on io.js, you can see the announcement [here](https://twitter.com/wjblankenship/status/575867637680369665)
+* [Node.js Advisory Board](https://nodejs.org/about/advisory-board/) are talking about the [io.js/Node.js reconcilitation proposal](https://github.com/iojs/io.js/issues/978), you can check the meeting minutes [here](https://github.com/joyent/nodejs-advisory-board/blob/master/meetings/2015-03-09/minutes.md#nodejsiojs-reconciliation-bb)
```
Comment (e-jigsaw) [inline weekly-updates/weekly-update.2015-03-13.md]: typo

reconcilitation -> reconciliation

**Comment by yosuke-furukawa** [comment]
LGTM

**Comment by dotproto** [comment]
As I understand it we can't publish this post to the "Node & Javascript" publication on Medium because we need @mikeal to approve it, but he's unfortunately at the moment.

@julianduque suggested we publish to the iojs Medium account. +1 to that idea from me.

IMO we should be reliable and consistent with our communications. Later we can re-publish the article to Node & Javascript and have the old article link to the new location (if necessary).

**Comment by julianduque** [comment]
https://medium.com/@iojs/io-js-week-of-march-13th-e3024cc66802

**Pull request merged**